tests/validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -31,16 +31,16 @@ def cattle_url():
 
 
 @pytest.fixture(autouse=True, scope='session')
-def cleanup(client):
+def cleanup():
     to_delete = []
-    for i in client.list_instance(state='running'):
+    for i in _admin_client().list_instance(state='running'):
         try:
```

Comment:
James, is there a reason why we've switched from using client to admin_client here?